socket.io.patch.js

```diff
@@ -1,3 +1,9 @@
+// EventEmitter has been removed from process in node >= 7
+// https://github.com/nodejs/node/commit/62b544290a075fe38e233887a06c408ba25a1c71
+if(process.versions.node.split('.')[0] >= 7) {
+  process.EventEmitter = require('events')
+}
+
```

Comment:
I would have liked to include this piece under the `if (io.version === "0.9.16")` block, but `process.EventEmitter` needs to be available before `require("socket.io")`

Reply:
I guess `if (!process.EventEmitter) { process.EventEmitter = ... }` is an alternative that doesn't depend on checking the version string.